src/test/regress/expected/metadata_sync_helpers.out

```diff
@@ -38,7 +37,11 @@ BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;
 
 	SET application_name to 'citus_internal gpid=10000000001';
 	SELECT citus_internal_add_partition_metadata ('test'::regclass, 'h', 'col_1', 0, 's');
-ERROR:  This is an internal Citus function can only be used in a distributed transaction
+ citus_internal_add_partition_metadata
```

Comment:
It's not clear to me why this change in behavior is needed for the current change.

Reply:
Because now EnsureCitusInitiatedOperation doesn't enforce `!IsCoordinator` anymore as we want to run some of those citus_internal UDFs in coordinator too.